src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1276,7 +1276,7 @@
     <value>Duplicate constraint '{0}' for type parameter '{1}'</value>
   </data>
   <data name="ERR_ClassBoundNotFirst" xml:space="preserve">
-    <value>The class type constraint '{0}' must come before any other constraints</value>
```

Comment:
>class  [](start = 15, length = 6)

I do not think we should make his change. This error is specifically about class types (record classes are classes), not about interface types, for example.